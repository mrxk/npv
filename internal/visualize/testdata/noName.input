---
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
spec:
  ingress:
  - from:
    - podSelector:
        matchLabels:
          one: one
    - namespaceSelector: {}
    ports:
    - port: 1111
      protocol: TCP
    - port: 2222
      protocol: TCP
  podSelector:
    matchLabels:
      one: one
  policyTypes:
  - Ingress