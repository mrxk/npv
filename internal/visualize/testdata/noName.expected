@startuml
left to right direction
frame Pods {
component "Name: \lNamespace: \lMatch Labels:\l    one: one\l" as oneone {
    port "1111 (TCP)" as _ALL_TCP1111port
    port "2222 (TCP)" as _ALL_TCP2222port
    port "1111 (TCP)" as oneoneTCP1111port
    port "2222 (TCP)" as oneoneTCP2222port
}
}
frame Ingress {
component "Namespace:\l    All\l" as _ALL__i {
    portout " " as _ALL_ingressportout
}
component "Pod:\l    Match Labels:\l        one: one\l" as oneone_i {
    portout " " as oneoneingressportout
}
}
_ALL_ingressportout --down[#green]--> _ALL_TCP1111port
_ALL_ingressportout --down[#green]--> _ALL_TCP2222port
oneoneingressportout --down[#green]--> oneoneTCP1111port
oneoneingressportout --down[#green]--> oneoneTCP2222port
frame Egress {
}
@enduml
